File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/keys/model/NotifyParams.java
package org.ngbp.jsonrpc4jtestharness.rpc.keys.model;

import java.util.List;

public class NotifyParams {
    public String msgType;
    public List<Timeout> timeout;
    public class Timeout{
        public String key;
        public Integer time;
    }
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/keys/model/NotifyParams.kt
package org.ngbp.jsonrpc4jtestharness.rpc.keys.model

data class NotifyParams (
    var msgType: String? = null,
    var timeout: MutableList<Timeout?>? = null
)

data class Timeout (
    var key: String? = null,
    var time: Int? = null
)
